src/helpers/createBlob.js

```diff
@@ -0,0 +1,8 @@
+export default (str, mime) => {
+  const string = typeof str === 'object' ? JSON.stringify(str) : str;
+  if (string === null) return '';
+  const blob = new Blob([string], {
+    type: mime
+  });
+  return window.URL.createObjectURL(blob);
+};
```

Comment:
can you check how we do it for web file download and may be use the same function if it is similiar?

Reply:
Oh this refactors that. Both the keystore download uses this function now